docs/testingprocedures.md

```diff
@@ -0,0 +1,29 @@
+#Perform this checklist every test day: 
+
+##Before leaving
+
+###Packing list 
+[copy from list from the excell file in the grabcad]
+
+###Software Checklist
+*Update and build code
+*Verify all sensors output data
+*Verify thrusters spin given command
+*Verify kill (soft an hard)
+
+
+##At testing site
+
+*Get wall power to powerstrip
+*Setup and connect to Network Box
+*Roll put tether and connect it ot network box **(DO NOT USE POE)**
+*Connect Sub to to tether *(NOT POE)*
+*Connect battery alarm, power on sub
+*SSH into sub
+*Start tmux, write code
+*Grease O-rings with Molykote 55 everytime a pressure vessel is closed
+*Make sure the pneumatic tubes are inserted correctly **(DO NOT FLOOD THE VALVE BOX)**
+*Person getting into the pool must do backflip
+*Deploy sub (check for bubbles, make sure buoyancy is correct)
+*Verify odometry
```

Comment:
I don't think this will be rendered correctly as it is invalid markdown. Try running `scripts/build_docs` then `scripts/display_docs` so you can verify the docs locally